include/picongpu/fields/incidentField/profiles/DispersiveLaser.hpp

```diff
@@ -218,7 +218,7 @@ namespace picongpu
                             float_X center = Unitless::SD * (Omega - Unitless::w)
                                 + SPEED_OF_LIGHT * alpha * focusPos / (Unitless::W0 * Unitless::w);
 
-                            // inverse radius of curvature of the beam's  wavefronts
+                            // inverse radius of curvature of the pulse's  wavefronts
```

Comment:
could you please remove the two spaces here

Reply:
Done.